content/W2015/W15-blog-hollyxu.md

```diff
@@ -0,0 +1,51 @@
+Title: Get to Know: Holly Xu
+Date: 2015-01-23 19:00
+Category: Blog
+Tags: get to know
+Slug: get-to-know-holly-xu
+Author: Julia Nguyen
+Summary: This week I sat down with Holly Xu, a fourth year student to talk about her transition into CS from accounting, how robotics club boosted her confidence, and the importance of independence.
+
+There's a recurring theme in these interviews - taking risks. Overcoming self-esteem is a huge part of being a woman in computing. This week I sat down with Holly Xu, a fourth year student to talk about her transition into CS from accounting, how robotics club boosted her confidence, and the importance of independence.
```

Comment:
interviews - taking risks <-- this should be either an `&ndash;` with spaces or an `&mdash;` without, but not a hyphen
Also, same fixes as my comment above for the second line